testgrid/config.yaml

```diff
@@ -312,10 +312,6 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-device-plugin-gpu-beta
   alert_stale_results_hours: 24
   num_failures_to_alert: 12 # Runs every 2h. Alert when it's been failing for a day.
-- name: ci-kubernetes-e2e-gce-gpu-stable2
```

Comment:
there's another reference to `ci-kubernetes-e2e-gce-gpu-stable2` if you grep

Reply:
done